Application has thrown an uncaught exception and is terminated:
Error: listen EADDRINUSE
    at errnoException (net.js:878:11)
    at Server._listen2 (net.js:1016:14)
    at listen (net.js:1038:10)
    at Server.listen (net.js:1104:5)
    at Object.<anonymous> (D:\Work\.NET Tests\nodejs\echoapphb\echoer\server.js:9:4)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
